ansible_collections/arista/avd/plugins/plugin_utils/schema/store.py

```diff
@@ -127,12 +127,8 @@ def _resolve_schema(schema: dict, store: dict) -> dict:
     .schemaresolver performs inplace update of the argument so we give it a copy of the existing schema.
     """
     resolved_schema = deepcopy(schema)
-    schemaresolver = AvdSchemaResolver(resolved_schema, store)
-    resolve_errors = schemaresolver.iter_errors(resolved_schema)
-    for resolve_error in resolve_errors:
-        if isinstance(resolve_error, Exception):
-            # TODO: Raise multiple errors or abstract them
-            raise resolve_error
+    schemaresolver = AvdSchemaResolver(schema["$id"], store)
+    schemaresolver.resolve(resolved_schema)
```

Comment:
I am not sure while reading the code where the error handling went. Are we still raising on resolving errors?

Reply:
no change in behavior and this is handled inside resolve